src/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -415,6 +418,16 @@ class LocalTrainingService implements TrainingService {
         const filepath: string = path.join(directory, `parameter_${hyperParameters.index}.cfg`);
         await fs.promises.writeFile(filepath, hyperParameters.value, { encoding: 'utf8' });
     }
+
+    private generateSequenceId(): number {
+        return this.trialSequenceId++;
+    }
+
+    private async writeSequenceIdFile(trialJobId: string): Promise<void> {
```

Comment:
I suggest to put sequenceId into LocalTrialJobDetail, and make it as readonly, rather than generating it when calling writeSequenceIdFile function

Reply:
Thanks,  updated per your comments, sequenceId added into common TrialJobDetail.